### PR TITLE
Fix selectstream

### PR DESCRIPTION
**Pull request by smialko**
It was impossible to select a measurements stream on graph or map view, the view went back to the dashboard. 
Task: https://trello.com/c/RnoCe7eH/124-selecting-streams-on-map-graph-view

### CODE REVIEW

**Review by jankrzempek — APPROVED** [review]
Cool!

File: AirCasting/CoreData/SessionEntity.swift [hunk 1]
```diff
@@ -77,6 +77,28 @@ public class SessionEntity: NSManagedObject, Identifiable {
...
+            let bIdx = defaultSortedABStreams.firstIndex(of: b)
+            
+            // If 'A' stream is unknown, B should be first.
+            guard let aIdx = aIdx else { return false }
+            // If 'B' stream is unknown, A should be first.
+            guard let bIdx = bIdx else { return true }
+            
+            // If 'A' and 'B' streams are known,
+            // preserve order of 'defaultSortedABStrams'
+            return aIdx < bIdx
```
Comment (pawelgil) [inline AirCasting/CoreData/SessionEntity.swift:98]: well done with the comments, those make a world of difference in readability for such algorithms

File: AirCasting/SessionViews/ABMeasurementsView.swift [hunk 2]
```diff
@@ -16,14 +16,8 @@ struct ABMeasurementsView: View {
...
-        let allStreams = [session.pm1Stream,
-                          session.pm2Stream,
-                          session.pm10Stream,
-                          session.FStream,
-                          session.HStream]
-        
-        let toShow = allStreams.compactMap { $0 }
-        return toShow
+        guard let streams = session.sortedStreams else { return [] }
+        return streams
```
Comment (pawelgil) [inline AirCasting/SessionViews/ABMeasurementsView.swift:20]: we can use `??` operator here:
```Swift
 return session.sortedStreams ?? []
```

File: AirCasting/SessionViews/SessionCartView.swift [hunk 3]
```diff
@@ -51,20 +53,46 @@ struct SessionCartView: View {
...
         .font(Font.moderate(size: 13, weight: .regular))
         .foregroundColor(.aircastingGray)
         .padding()
         .background(
-            Color.white
-                .shadow(color: Color(red: 205/255, green: 209/255, blue: 214/255, opacity: 0.36), radius: 9, x: 0, y: 1)
+            Group {
+                Color.white
+                    .shadow(color: Color(red: 205/255, green: 209/255, blue: 214/255, opacity: 0.36), radius: 9, x: 0, y: 1)
+                NavigationLink(
```
Comment (pawelgil) [inline AirCasting/SessionViews/SessionCartView.swift:69]: would be nice to move those navigation links to separate vars so it's easier to read thru this layout

**Review by pawelgil — APPROVED** [review]

**Pull request merged**